libqtile/layout/tile.py

```diff
@@ -34,35 +34,57 @@
 
 
 class Tile(_SimpleLayoutBase):
+    """A layout with two stacks of windows dividing the screen
+
+    The Tile layout divides the screen_rect horizontally into two stacks. The
+    maximum amount of "master" windows can be configured; surplus windows will
+    be displayed in the slave stack on the right.
+    Within their stacks, the windows will be tiled vertically.
+    The windows can be rotated in their entirety by calling up() or down() or,
+    if shift_windows is set to True, individually.
+    """
+
     defaults = [
         ("border_focus", "#0000ff", "Border colour for the focused window."),
         ("border_normal", "#000000", "Border colour for un-focused windows."),
         ("border_width", 1, "Border width."),
         ("name", "tile", "Name of this layout."),
         ("margin", 0, "Margin of the layout"),
+        ("ratio", 0.618,
+            "Width-percentage of screen size reserved for master windows."),
+        ("masterWindows", 1,  # TODO: masterWindows should be named master_windows
```

Comment:
Would renaming this be ok?

Reply:
It wasn't even documented, so let's rename it and add a line in the change log.

Reply:
That sounds good.